### PR TITLE
Update Monica to v3.7.0

### PR DESCRIPTION
**Pull request by asbiin**
Update Monica to [v3.7.0](https://github.com/monicahq/monica/releases/tag/v3.7.0).

### CODE REVIEW

**Comment by asbiin** [comment]
@tianon I did change the default branch of https://github.com/monicahq/docker from `master` to `main`, maybe it was not a good idea?

**Comment by tianon** [comment]
Naw, that's fine, you just need to add `GitFetch: refs/heads/main` next to your `GitRepo:` field.

File: library/monica [hunk 1]
```diff
@@ -1,18 +1,18 @@
-# This file is generated via https://github.com/monicahq/docker/blob/437356f9ed4fe4e22f12fc38451ae494f971175c/generate-stackbrew-library.sh
+# This file is generated via https://github.com/monicahq/docker/blob/6d256f443e7cc900b48c65a3e90472a6ef42c34c/generate-stackbrew-library.sh
 Maintainers: Alexis Saettler <alexis@saettler.org> (@asbiin)
 GitRepo: https://github.com/monicahq/docker.git
```
Comment (tianon) [inline library/monica:3]:
```suggestion
GitRepo: https://github.com/monicahq/docker.git
GitFetch: refs/heads/main
```

**Comment by asbiin** [comment]
Thank you @tianon !

**Pull request merged**